shared/modules/types.go

```diff
@@ -1,6 +1,228 @@
-package types
+package modules
 
-// TODO(pocket/issues/163): Move this into utility
+import (
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+// This file contains the minimum shared structures (GenesisState) and the many shared interfaces the modules implement
+// the main purpose of this structure is to ensure the ownership of the
+
+type GenesisState struct {
+	PersistenceGenesisState PersistenceGenesisState `json:"persistenceGenesisState"`
```

Comment:
The json tags in `BaseConfig` are `snake_case` but `camelCase` here.

Please switch these to `snake_case` to stay consistent